pm_icecon/nt/params/gradient_thresholds.py

```diff
@@ -0,0 +1,101 @@
+"""Gradient thresholds for the nasateam weather filter.
+"""
+from pm_tb_data._types import NORTH, SOUTH
+
+from pm_icecon.nt._types import NasateamGradientRatioThresholds
+from pm_icecon.errors import UnexpectedSatelliteError
+
+# These thresholds defined specifically for the CDR. These differ from thoes
+# defined by Goddard.
```

Comment:
These gradient thresholds are specific to the CDR, so I added a note here and made the variable and function names explicit.